tests/unit/services/file-manager-test.js

```diff
@@ -97,69 +29,36 @@ moduleFor('service:file-manager', 'Unit | Service | file manager', {
 
         // FactoryGuy setup
         manualSetup(this.container);
-        mockSetup();
-    },
-    afterEach() {
-        mockTeardown();
     }
 });
 
 test('getContents sends valid waterbutler request', function (assert) {
-    assert.expect(4);
-    let service = this.subject();
-    let file = FactoryGuy.make('file');
-    let done = assert.async();
+    assert.expect(2);
 
-    let request = {
+    const service = this.subject();
+    const file = FactoryGuy.make('file');
+
+    const request = {
         url: file.get('links').download,
         settings: {method: 'GET'},
         headers: {Authorization: `Bearer ${fakeAccessToken}`}
     };
-    let response = {
-        status: 200,
-        data: 'file contents here'
-    };
-    mockWaterbutler(assert, request, response);
 
-    service.getContents(file).then(function (data) {
-        assert.equal(data, response.data);
-        done();
-    }).catch(function () {
-        assert.ok(false, 'promise should not reject on success');
-        done();
-    });
-});
+    const stub = this.stub(service, '_waterbutlerRequest');
 
-test('getContents passes along error', function (assert) {
-    assert.expect(4);
-    let service = this.subject();
-    let file = FactoryGuy.make('file');
-    let done = assert.async();
+    service.getContents(file);
 
-    let request = {
-        url: file.get('links').download,
-        settings: {method: 'GET'},
-        headers: {Authorization: `Bearer ${fakeAccessToken}`}
-    };
-    let response = {
-        status: 404
-    };
-    mockWaterbutler(assert, request, response);
+    assert.ok(stub.calledOnce, '_waterbutlerRequest was called once');
 
-    service.getContents(file).then(function () {
-        assert.ok(false, 'promise should reject');
-        done();
-    }).catch(function () {
-        assert.ok(true, 'promise rejects on error');
-        done();
-    });
+    assert.ok(stub.calledWithExactly('GET', request.url, {}), '_waterbutlerRequest was called with correct parameters');
```

Comment:
Your changes make sense, and are more like actual unit tests, but there are no longer tests for the methods you consistently stub, like `_waterbutlerRequest`.